test/csuite/timestamp_abort/main.c

```diff
@@ -489,7 +490,7 @@ backup_delete_old_backups(int retain)
     DIR *d;
     size_t len;
     int count, i, indexes[256], last_full, ndeleted;
-    char buf[256];
+    char buf[256], new[256];
```

Comment:
I'd recommend that we use a different name, as `new` is a reserved word in C++, especially as there is an interest in making our C code C++-compatible in the future.

Reply:
Fixed.